src/class/CgiManager/CgiManager.hpp

```diff
@@ -0,0 +1,70 @@
+
+#ifndef CGI_MANAGER_HPP
+#define CGI_MANAGER_HPP
+
+#include <BasicHTTPRequest/BasicHTTPRequest.hpp>
+#include <Path/Path.hpp>
+#include <Pipe/Pipe.hpp>
+#include <PollManager/PollManager.hpp>
+#include <Tab/Tab.hpp>
+#include <converter/converter.hpp>
+
+#include <cstring>
+#include <iostream>
+#include <string>
+#include <sys/wait.h>
+#include <unistd.h>
+
+#ifdef TEST_ON
+#define private public
+#define protected public
+#endif
+
+class CgiManager
```

Comment:
this version of the class is much better. it is clear and split into clever small parts